src/index.ts

```diff
@@ -17,6 +22,35 @@ import * as https from 'https'
 import * as fs from 'fs'
 import protectOathkeeper from './middleware/oathkeeper'
 import protectSimple from './middleware/simple'
+import bodyParser from 'body-parser'
+import csrf from 'csurf'
+import session from 'express-session'
+
+const csrfProtection = csrf({ cookie: true })
+
+const publicEndpoint = new PublicApi(
+  new Configuration({ basePath: config.kratos.public })
+)
+const protectProxy = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
This code is implemented here https://github.com/ory/kratos-selfservice-ui-node/blob/master/src/middleware/simple.ts - is there a particular reason for not using that instead?

Reply:
Agreed, this is now dead code.  Removed.